Websites/webkit.org/wp-content/themes/webkit/team.php

```diff
@@ -17,7 +17,11 @@
         --expertise-shadow: 0px 5px 10px 0px hsla(0, 0%, 100%, 0.1);
         --hover-text-color: hsl(0, 0%, 100%);
     }
+    img {
```

Comment:
That seems error-prone for the future, maybe add a class name on the Github icon and target that?

Reply:
It seems to be the existing convention in the file, also for `em` and `span`.

Reply:
They do have `li` in front (though I don't find that safe enough either tbh). As it stands, your rule could target anything including images in the navbar or footer. Personally I'd just add a `github-icon` class name from the JS.